src/TracksConfigure.tsx

```diff
@@ -80,7 +79,9 @@ const TracksConfigure: React.FC<
         localVideoTrack.close()
         localVideoTrack.stop()
       }
-      callbacks?.EndCall && callbacks.EndCall()
+      if (error) {
+        callbacks?.EndCall && callbacks.EndCall()
+      }
```

Comment:
I guess if you do it with a conditional, this is clearer:
```suggestion
      if (error && callbacks?.EndCall) {
         callbacks.EndCall()
      }
```